app/services/doc_auth_router.rb

```diff
@@ -219,17 +219,7 @@ def self.notify_exception(exception, custom_params = nil)
     end
   end
 
-  #
-  # The `acuant_simulator` config is deprecated. The logic to switch vendors
-  # based on its value can be removed once FORCE_ACUANT_CONFIG_UPGRADE in
```

Comment:
Should we remove the `FORCE_ACUANT_CONFIG_UPGRADE` too now and clean up that branching?

Reply:
Nevermind, Github just collapsed that file.... ignore this